physionet-django/oauth/views.py

```diff
@@ -2,6 +2,6 @@
 from oauth2_provider.views.generic import ProtectedResourceView
 
 
-class ApiEndpoint(ProtectedResourceView):
+class hello(ProtectedResourceView):
```

Comment:
Did this happen intentionally?

Reply:
Yup - I changed it to Hello to keep the view name and the URL endpoint have the same name.